content/finance/spending-company-money.md

```diff
@@ -42,26 +42,54 @@ The estimated spend for interns is **$925\*** and includes only the electronic i
+
+Different specs are required repending on your role, please see below for necessary specs corresponding to your role.
+
+### Customer Engineering team laptops:
+
+- Apple MacBook Pro - 14.2" M1 Pro - 16 GB RAM, 1 TB SSD
+  - [See product in CDW for reference](https://www.cdw.com/product/apple-macbook-pro-14.2-m1-pro-16-gb-ram-1-tb-ssd-us/6738975?enkwrd=6738975)
+- Apple MacBook Pro - 16.2" M1 Pro - 16 GB RAM, 1 TB SSD
+  - [See product in CDW for reference](https://www.cdw.com/product/apple-macbook-pro-16.2-m1-pro-16-gb-ram-1-tb-ssd-us/6738957?enkwrd=6738957)
 - Any other laptop that is of equal or lesser value than the pre-approved laptop.
 
-**Laptop (product team):**
+### Product team laptops:
 
-- 13" MacBook Pro, 2.3 GHz (4.1 GHz turbo) 4-core (upgrade), 32 GB (upgrade), 512 GB SSD. If you need to update any specs, please reach out to @sourcegraph/people-ops for approval.
+- Apple MacBook Pro - 16.2" M1 Max - 32 GB RAM, 512 GB SSD
+  - [See product in CDW for reference]()
 - Any other laptop that is of equal or lesser value than the pre-approved laptop.
 
-**Laptop (all other teams):**
+### Engineering team laptops:
 
-- 13" MacBook Pro, 2.0 GHz (3.8 GHz turbo) Intel Core i5 processor, 16 GB, 512 GB SSD. If you need to update any specs, please reach out to @sourcegraph/people-ops for approval.
+- Apple MacBook Pro - 16.2" M1 Max - 64 GB RAM, 1 TB SSD
+  - [See product in CDW for reference](https://www.cdw.com/product/apple-mbp-16-m1m-10c32c-64-1tb-sg/6740562?enkwrd=6740562)
+- Apple MacBook Pro - 16.2" M1 Max - 32 GB RAM, 1 TB SSD
+  - [See product in CDW for reference](https://www.cdw.com/product/apple-macbook-pro-16.2-m1-max-32-gb-ram-1-tb-ssd-us/6738961?enkwrd=6738961)
 - Any other laptop that is of equal or lesser value than the pre-approved laptop.
 
-**Other computer:**
+### Customer Support team laptops:
+
+- Apple MacBook Pro - 16.2" M1 Max - 32 GB RAM, 1 TB SSD
+  - [See product in CDW for reference](https://www.cdw.com/product/apple-macbook-pro-16.2-m1-max-32-gb-ram-1-tb-ssd-us/6738961?enkwrd=6738961)
+
+### All other teams laptops:
+
+- Apple MacBook Pro - 14.2" M1 Pro - 16 GB RAM, 512 GB SSD
+  - [See product in CDW for reference](https://www.cdw.com/product/apple-macbook-pro-14-apple-m1-pro-chip-16-gb-ram-512-gb-ssd-spac/6738971?enkwrd=6738971)
+- Any other laptop that is of equal or lesser value than the pre-approved laptop.
+
+If you need to update any specs, please reach out to your manager and [People Ops](mailto:people-ops@sourcegraph.com) for approval.
+
+### Other computer
 
-- If you want something different (such as a desktop computer), that is obviously fine. If it's more expensive than the laptops listed above, please check with @sourcegraph/people-ops first.
+- If you want something different (such as a desktop computer), that is obviously fine. If it's more expensive than the laptops listed above, please check with your manager and [People Ops](mailto:people-ops@sourcegraph.com) first.
```

Comment:
@InesRoitman @nickyvm Do we want to mention a a desktop computer as an option? Not to say we wouldn't accommodate, but ideally these would be one off requests.

Reply:
Yeah probably best to simply just leave it as: if you want something different please contact tech-ops but if it is more expensive than the laptops listed above please have managerial or people ops approval before requesting